pkg/apis/vpc/v1alpha1/subnetconnectionbindingmap_types.go

```diff
@@ -0,0 +1,61 @@
+package v1alpha1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +kubebuilder:validation:XValidation:rule="!(has(self.targetSubnetSetName) && has(self.targetSubnetName))",message="Only one of targetSubnetSetName or targetSubnetName can be specified"
```

Comment:
does this expression also true if there is no targetSubnetSetName and targetSubnetName?

Reply:
Good catch, updated.